### PR TITLE
chore: improve app typography

### LINKED ISSUE #21426: Improve default app typography settings
1. Make `system default` our option for the default theme (`Modern`)
2. Put `Modern` at the top of the theme list
3. ~Rename `Modern` to `Default`~

All existing applications should continue to use whatever is in their theme (e.g. `Nunito`), all new applications should be built on `system default`.

### PR DESCRIPTION
**Pull request by jsartisan**
This PR changes:
1. Changes the order of default theme in theme list
2. sets the default font from nunito to System default

Fixes #21426

## Type of change

- Bug fix 

## How Has This Been Tested?

- Manual
- Cypress

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/server/appsmith-server/src/main/resources/system-themes.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [
   {
-    "name": "Classic",
-    "displayName": "Classic",
+    "name": "Default",
+    "displayName": "Modern",
```
Comment (KelvinOm) [inline app/server/appsmith-server/src/main/resources/system-themes.json:4]: @ichik Should we change displayName to `Default`?

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Git/GitWithTheming/GitWithTheming_spec.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import * as _ from "../../../../../support/Objects/ObjectsCore";
 const commonlocators = require("../../../../../locators/commonlocators.json");
 
 describe("Git with Theming:", function () {
-  const backgroudColorMaster = "rgb(85, 61, 233)";
+  const backgroudColorMaster = "rgb(22, 163, 74)";
```
Comment (KelvinOm) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Git/GitWithTheming/GitWithTheming_spec.js:6]: Why did the color change in the test?

File: app/server/appsmith-server/src/main/resources/system-themes.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 [
   {
-    "name": "Classic",
-    "displayName": "Classic",
+    "name": "Default",
```
Comment (KelvinOm) [inline app/server/appsmith-server/src/main/resources/system-themes.json:3]: How can we be sure that the font for the created apps will remain `Nunito`?

Comment (jsartisan) [reply app/server/appsmith-server/src/main/resources/system-themes.json:3]: that is hard to test to cypress. Let me test it manually. The process is like this:

1. Create a app on release
2. open the same on your DP with the same credentials. The app should not fail or should have desired output you were expecting.

File: app/server/appsmith-server/src/main/resources/system-themes.json (hunk 1 above)
Comment (jsartisan) [reply app/server/appsmith-server/src/main/resources/system-themes.json:4]: I think it was decided that all themes have a distinct name. The list of actions didn't have this item ( removed by vadim ). so I didn't update it.

Comment (ichik) [reply app/server/appsmith-server/src/main/resources/system-themes.json:4]: Keeping modern for now

**Comment by kamakshibhat-appsmith** [comment]
Observations:
On importing an application which had Nunito as font , the font is getting set as System Default @jsartisan

**Comment by jsartisan** [comment]
@dilippitchika need your help on this. Will need someone from backend team to unblock us. It needs a backend migration.

**Comment by somangshu** [comment]
@jsartisan @KelvinOm closing this PR for now

**Pull request closed without merging**